src/api/api.js

```diff
@@ -27,10 +49,13 @@ export function getLocationInfo(locationId) {
  *
  * @param {String} timeAggregation The aggregation of the data (one of day, month, year)
  * @param {String} locationId The location to query (e.g., nauswaseattle)
+ * @param {Options} object Options with startDate and endDate
  * @return {Promise} A promise after the get request was made
  */
-export function getLocationTimeSeries(timeAggregation, locationId) {
-  return get(`/locations/${locationId}/time/${timeAggregation}/metrics`)
+export function getLocationTimeSeries(timeAggregation, locationId, options = {}) {
+  const params = getDateRangeParams(timeAggregation, options);
+  const paramString = params.length > 0 ? `?${params.join('&')}` : '';
+  return get(`/locations/${locationId}/time/${timeAggregation}/metrics${paramString}`)
```

Comment:
You don't have to do this paramString stuff. `get` takes the query parameters as its second arguments. Just pass it in as an object.